src/liboslexec/opmatrix.cpp

```diff
@@ -198,20 +198,24 @@ osl_get_inverse_matrix(void* sg_, void* r, const char* to)
 // Implemented by the renderer
```

Comment:
For opmatrix, would some of this optix specific stuff go away, if osl_get_matrix and osl_get_inverse_matrix were NOT implemented by the renderer and just use the bitcode based free function renderer services.  Seems like alot of this might be because of the call out to optix then wanting optix to call back in.  Now the problem in general may still exist, but not sure we need all of these functions to be callable from the renderer side.  In general there might be a set of osl functions meant to be called by free renderer services, and perhaps they should be explicitly defined someplace with a different prefix building scheme (compatible with Optix).